core/templates/dev/head/domain/collection/GuestCollectionProgressService.js

```diff
@@ -42,8 +42,8 @@ oppia.factory('GuestCollectionProgressService', [
 
     var recordCompletedExploration = function(collectionId, explorationId) {
       var guestCollectionProgress = loadGuestCollectionProgress();
-      if (guestCollectionProgress.addCompletedExplorationId(
-        collectionId, explorationId)) {
+      if (guestCollectionProgress
+        .addCompletedExplorationId(collectionId, explorationId)) {
```

Comment:
This is getting clumsy; consider just pulling this out into a separate variable.

Reply:
Hi @seanlip, is this fine?  
```javascript
var completedExplorationIdHasBeenAdded = (
        guestCollectionProgress
          .addCompletedExplorationId(collectionId, explorationId));
```

Thanks!

Reply:
Yes, that looks good, assuming you intend to indent the first line more. You could also do

```
      var completedExplorationIdHasBeenAdded = (
        guestCollectionProgress.addCompletedExplorationId(
          collectionId, explorationId));
```

Reply:
Done.